src/test/java/org/openrewrite/java/testing/junit5/UpdateTestAnnotationTest.java

```diff
@@ -106,23 +106,62 @@ public void test() {
         );
     }
 
+    @Test
```

Comment:
In test case above, can't inline even though single line ... and need to leave like that. So have to consider these situations.